wavefront/data_source_alert.go

```diff
@@ -217,82 +217,82 @@ func dataSourceAlertRead(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
-	if err := d.Set("display_expression", alert.DisplayExpression); err != nil {
+	if err := d.Set(displayExpressionKey, alert.DisplayExpression); err != nil {
 		return err
 	}
-	if err := d.Set("minutes", alert.Minutes); err != nil {
+	if err := d.Set(minutesKey, alert.Minutes); err != nil {
 		return err
 	}
-	if err := d.Set("resolve_after_minutes", alert.ResolveAfterMinutes); err != nil {
+	if err := d.Set(resolveAfterMinutesKey, alert.ResolveAfterMinutes); err != nil {
 		return err
 	}
-	if err := d.Set("notification_resend_frequency_minutes", alert.NotificationResendFrequencyMinutes); err != nil {
+	if err := d.Set(notificationResendFrequencyMinutesKey, alert.NotificationResendFrequencyMinutes); err != nil {
 		return err
 	}
-	if err := d.Set("severity", alert.Severity); err != nil {
+	if err := d.Set(severityKey, alert.Severity); err != nil {
 		return err
 	}
-	if err := d.Set("severity_list", alert.SeverityList); err != nil {
+	if err := d.Set(severityListKey, alert.SeverityList); err != nil {
 		return err
 	}
-	if err := d.Set("status", alert.Status); err != nil {
+	if err := d.Set(statusKey, alert.Status); err != nil {
 		return err
 	}
-	if err := d.Set("tags", alert.Tags); err != nil {
+	if err := d.Set(tagsKey, alert.Tags); err != nil {
 		return err
 	}
 	if err := d.Set(runbookLinksKey, alert.RunbookLinks); err != nil {
 		return err
 	}
-	if err := d.Set("can_view", alert.ACL.CanView); err != nil {
+	if err := d.Set(canViewKey, alert.ACL.CanView); err != nil {
 		return err
 	}
-	if err := d.Set("can_modify", alert.ACL.CanModify); err != nil {
+	if err := d.Set(canModifyKey, alert.ACL.CanModify); err != nil {
 		return err
 	}
-	if err := d.Set("process_rate_minutes", alert.CheckingFrequencyInMinutes); err != nil {
+	if err := d.Set(processRateMinutesKey, alert.CheckingFrequencyInMinutes); err != nil {
 		return err
 	}
-	if err := d.Set("evaluate_realtime_data", alert.EvaluateRealtimeData); err != nil {
+	if err := d.Set(evaluateRealtimeDataKey, alert.EvaluateRealtimeData); err != nil {
 		return err
 	}
-	if err := d.Set("include_obsolete_metrics", alert.IncludeObsoleteMetrics); err != nil {
+	if err := d.Set(includeObsoleteMetricsKey, alert.IncludeObsoleteMetrics); err != nil {
 		return err
 	}
-	if err := d.Set("failing_host_label_pairs", flattenHostLabelPairs(alert.FailingHostLabelPairs)); err != nil {
+	if err := d.Set(failingHostLabelPairsKey, flattenHostLabelPairs(alert.FailingHostLabelPairs)); err != nil {
 		return err
 	}
-	if err := d.Set("in_maintenance_host_label_pairs", flattenHostLabelPairs(alert.InMaintenanceHostLabelPairs)); err != nil {
+	if err := d.Set(inMaintenanceHostLabelPairsKey, flattenHostLabelPairs(alert.InMaintenanceHostLabelPairs)); err != nil {
 		return err
 	}
 	if err := d.Set(alertTriageDashboardsKey, parseAlertTriageDashboards(alert.AlertTriageDashboards)); err != nil {
 		return err
 	}
-	return d.Set("process_rate_minutes", alert.CheckingFrequencyInMinutes)
+	return nil
```

Comment:
I delete this `Set` because it was duplicated - see `line 277` above. I also changed it to return a nil explicitly at the end of the function (rather than the return of the last `Set` called), to keep the layout consistent with the rest of the calls to `Set`.

Reply:
Huh, that's interesting. Doesn't seem like a situation where side effects would require calling twice. Interesting that it was put in a return statement as well, if it fails does it change the output?

As long as no one tested any dependency here, it's a little on past devs if this breaks anything ...